examples/complete/main.tf

```diff
@@ -1,3 +1,102 @@
 ##############################################################################
-# Complete example
+# Resource group
 ##############################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.0"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+##############################################################################
+# Key Protect All Inclusive
+##############################################################################
+
+locals {
+  key_ring_name = "en-key-ring"
+  key_name      = "${var.prefix}-en"
+}
+module "key_protect_all_inclusive" {
+  source                    = "terraform-ibm-modules/key-protect-all-inclusive/ibm"
+  version                   = "4.2.0"
+  resource_group_id         = module.resource_group.resource_group_id
+  region                    = var.kms_region
```

Comment:
no this is not what I was suggestion. set this to `region = var.region` and then when calling the event notification module in this example, set `kms_region = var.region`

Reply:
Ok, done